selfdrive/car/hyundai/values.py

```diff
@@ -569,6 +563,24 @@ class Buttons:
       b'\xf1\x00OSE LKAS AT EUR LHD 1.00 1.00 95740-K4100 W40',
     ],
   },
+  CAR.KIA_NIRO_HEV: {
+    (Ecu.engine, 0x7e0, None): [
+      b'\xf1\x816H6F4051\000\000\000\000\000\000\000\000',
+    ],
+    (Ecu.transmission, 0x7e1, None): [
+      b"\xf1\x816U3J2051\000\000\xf1\0006U3H0_C2\000\0006U3J2051\000\000PDE0G16NS2\xf4\'\\\x91",
+      b'\xf1\x816U3J2051\000\000\xf1\0006U3H0_C2\000\0006U3J2051\000\000PDE0G16NS2\000\000\000\000',
```

Comment:
We're getting both of these variants back intermittently, verified in his uploaded logs. I'm not sure what causes there to be trailing garbage vs trailing nulls, and I don't have the cycles to find out. Carrying both variants in the FW dict is the only thing I can really do in response, unless comma have some better idea what's happening here.

Reply:
known issue. we've been including both for now